packages/blade/src/components/Badges/_decisions/decisions.md

```diff
@@ -0,0 +1,77 @@
+# Badge <!-- omit in toc -->
+Badges are used to show small amount of color coded metadata, which are ideal for getting users attention.
+
+<img src="./badge-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Badge](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A417)
+
+
+## API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| children | `string` | `undefined` | The label to be rendered within the badge. | ✅ |
+| variant | `positive`,`negative`, `notice`, `information`, `neutral` | `neutral` | The variant of the badge to be rendered. |  |
+| contrast | `low`, `high` | `low` | The contrast of the badge to be rendered. |  |
+| size | `medium`, `small` | `medium` | The size of the badge to be rendered. |  |
+| icon | `Icon` | `undefined` | The Blade `Icon` component to be rendered for the badge. |  |
+| fontStyle | `italic`, `normal` | `normal` | The font style of the badge's font. |  |
+| fontWeight | `bold`, `regular` | `regular` | The font weight of the badge's font. |  |
```

Comment:
why do we need to expose this? 🤔

Reply:
Part of design, we need to let the consumers make it bold or italic